Tests/BackendIntegrationTests/LoadShedderIntegrationTests.swift

```diff
@@ -39,6 +39,7 @@ class LoadShedderStoreKit1IntegrationTests: BaseStoreKitIntegrationTests {
     }
 
     func testCanGetOfferings() async throws {
+        // JOSH HERE - add one for meta data
```

Comment:
Actually won't need to anymore after #2499 :) simply run the test and update the snapshot 🎉